cmd/ddev/cmd/commands_test.go

```diff
@@ -254,6 +254,17 @@ func TestCustomCommands(t *testing.T) {
 		assert.NoError(err, "expected to find command %s for app.Type=%s", c, app.Type)
 	}
 
+	// Craft CMS commands should only be available for type craftcms
+	app.Type = nodeps.AppTypeCraftCms
+	_ = app.WriteConfig()
+	_, _ = exec.RunHostCommand(DdevBin)
+	err = app.MutagenSyncFlush()
+	assert.NoError(err)
+	for _, c := range []string{"craft"} {
```

Comment:
Interesting strategy! Not entirely sure this is the place to solve this, but doesn't seem to violate any particular strategies.